lib/loader.js

```diff
@@ -54,6 +54,10 @@ Loader = function () {
     fail('LiveScript is missing! Try `npm install livescript`');
   };
 
+  var _isRelative = function (p) {
+    return p[0] === '.' || p[0] === '/';
```

Comment:
One of the advantages of using Jake over straight CLI commands is that it's cross-platform. This will not work on Windows. Fortunately we're making heavy use in Jake of the `utilities` module, which has some useful cross-platform filesystem functions: https://github.com/mde/utilities/blob/master/lib/file.js#L393